src/mesh/parallel/shiftedmetric.cxx

```diff
@@ -34,8 +34,6 @@ ShiftedMetric::ShiftedMetric(Mesh& m) : ParallelTransform(m), zShift(&m) {
         "ShiftedMetric usually requires the option TwistShift=true\n"
         "    Set ShiftWithoutTwist=true to use ShiftedMetric without TwistShift");
   }
-
-  cachePhases();
```

Comment:
I'm not overly keen on this, as it means calling this constructor results in a partly initialised object, and we should strive to avoid that as much as possible

Reply:
yes, good point. I think it's only `zlength` that `ShiftedMetric` needs from `Coordinates`. Would it be better to just pass `zlength` to the constructor?

Reply:
If that's all that's needed, I think that would be cleaner